src/core/server/metrics/logging/get_ops_metrics_log.ts

```diff
@@ -83,8 +83,7 @@ export function getEcsOpsMetricsLog(metrics: OpsMetrics) {
           usedInBytes: processMemoryUsedInBytes,
         },
       },
-      eventLoopDelay: eventLoopDelayVal,
-      eventLoopDelayHistogram: eventLoopDelayHistVals,
+      eventLoopDelay: process?.event_loop_delay_histogram,
```

Comment:
The percentiles hide a lot of detail, while one slow event loop won't usually cause further problems, we really need to know the max event loop delay when profiling/optimizing since one slow event loop can turn into 100 slow event loops when there's more load.